doctrinal-works-v2/original/11-chapter-11.adoc

```diff
@@ -534,8 +534,10 @@ and in the psalms concerning me;`' so the law and the prophets are fulfilled in
 and he is the end of them; who is the first and the last, the beginning and the ending,
 the amen.
 
+[.signed-section-signature]
 G+++.+++ F.
 
+[.signed-section-context-close]
 The 1st day of the 7th month, 1676.
 
 === Postscript.
```

Comment:
I had trouble with this too because it said postscript headers are not allowed when I tried to wrap it in the postscript stuff.

Reply:
yup, sometimes in non-letters just treating a postscript as a new section (with a heading, like here) is good, i think.  i believe i actually added the `===` here because i thought it would make sense this way.  so this is good. 👍